### DISCUSSION TITLE
Reports in SP 2.0 to be depracated

### DISCUSSION THREAD

**Discussion by michalaleksandrowicz** [denisneuf/python-amazon-ad-api#101] [General]
Hi,

The documentation says that the reports endpoint in SP 2.0 is gonna be deprecated:

"Deprecation notice: This endpoint will be deprecated on March 30, 2023. Use the [version 3 reporting endpoints](https://advertising.amazon.com/API/docs/en-us/offline-report-prod-3p) going forward."

![image](https://user-images.githubusercontent.com/102812050/209828646-3c252699-97ec-4f3b-bdb3-781af395dbaf.png)

I am currently writing a script to pull the reports automatically with this great library, however I am worried that it is not gonna work after March 30th. Will the library be updated to reflect the new endpoint? 

Thanks!

**Comment by denisneuf** [comment]
Hi Michael, don’t worry, is already supported:

https://github.com/denisneuf/python-amazon-ad-api/pull/92 <https://github.com/denisneuf/python-amazon-ad-api/pull/92>

How about you start to migrate your stuff using the new endpoint?

https://python-amazon-ad-api.readthedocs.io/en/latest/api/reports.html <https://python-amazon-ad-api.readthedocs.io/en/latest/api/reports.html>

Cheers,

Daniel

**Reply by michalaleksandrowicz** [reply]
Great, now I see, I was checking wrong section of the documentation. Btw, great work on the library, helps a lot amateurs like me!
